.eslintrc.json

```diff
@@ -47,6 +47,14 @@
         ],
         "react/prop-types": [
             "off"
+        ],
+        "comma-dangle": [
+            "error",
+            "never"
+        ],
+        "import/extensions": [
```

Comment:
This is not going to do what you expect. It is an invalid syntax for the file, and actually breaks the import/export rule. So it won't flag your files, but only because it's crashing.

`Configuration for rule "import/extensions" is invalid: 	Severity should be one of the following: 0 = off, 1 = warn, 2 = error (you passed '".js"'). `

Reply:
Set this to "off" per suggested course of action.